arduino-core/src/cc/arduino/contributions/ConsoleProgressListener.java

```diff
@@ -33,12 +33,19 @@
 
 public class ConsoleProgressListener implements ProgressListener {
   private String lastStatus = "";
+  private double lastProgress = 0.0;
 
   @Override
   public void onProgress(Progress progress) {
-    if (!lastStatus.equals(progress.getStatus())) {
+    // Reduce verbosity when running in console
+    String s = progress.getStatus().replaceAll("[0-9]", "");
+    double p = progress.getProgress();
```

Comment:
This seems like a bit of a convoluted way to reduce verbosity here, but I'm not familiar enough with this Progress stuff to offer a better suggestion.